app/models/docker_registry.rb

```diff
@@ -0,0 +1,63 @@
+# model that represents a docker registry
+class DockerRegistry < ActiveRecord::Base
+  belongs_to :certificate, polymorphic: true
+
+  validates :url, presence: true
```

Comment:
I would enforce uniqueness over the url

Reply:
It's on data level (on the migration) but I agree we can add it here too.